frontend/public/components/persistent-volume-claim.jsx

```diff
@@ -265,6 +274,10 @@ const Details_ = ({ flags, obj: pvc }) => {
               <dd data-test-id="pvc-status">
                 <PVCStatus pvc={pvc} />
               </dd>
+              <dt>Requested Capacity</dt>
+              <dd data-test="pvc-requested-capacity">
```

Comment:
Is this being used in a test? I'd only add the data-test attribute when we plan to use it.

Reply:
Yes, This is required for the clone testing.